docs/en/advanced/Performance-Tuning.md

```diff
@@ -5,7 +5,328 @@ nickname: Performance Tuning
+should be used in scenarios when files a continually being directly added to UFS (not via Alluxio).
+In this case, Alluxio can always check the UFS to see if there are new files available. Since more
+frequent loading of metadata is costly with respect to performance, it is recommended to keep the
+setting as infrequent as possible.
+
+Compared with the UFS sync feature, loading metadata only discovers new files from the UFS. It does
+not modify any updated metadata of files, such as updated timestamps, or deleted files.
+
+### Syncing UFS Metadata
+
+Loading UFS metadata is important for discovering new files from the UFS, but it does not handle any
+modifications to existing UFS files. Therefore, Alluxio enables syncing the UFS metadata to
+Alluxio's metadata. To sync the metadata for a particular path, the Alluxio master will query the
+UFS for the same path, and then update the corresponding Alluxio metadata to match the UFS
+information. Users can optionally configure Alluxio to "sync" metadata with the UFS when running any
+Alluxio command. The user defined configuration value alluxio.user.file.metadata.sync.interval
+controls how often Alluxio operations should sync with the UFS. This configuration parameter defines
+the interval of time between multiple UFS synchronizations. A value of -1 means no syncing will be
+performed. 0 means Alluxio will always sync with UFS for every Alluxio operation. Any positive time
+interval defines the interval in which Alluxio will not re-sync the specified path. For example,
+using a value of "5m" means Alluxio will sync a particular path at most every 5 minutes.
+
+Since syncing the UFS metadata is an expensive operation, it recommended to not sync very often.
+However, there may be scenarios when it is desirable to do so. A value of -1 will never sync, so
+that is appropriate for when all writes go to Alluxio, and not bypassed to UFS. In this scenario,
+Alluxio will always know about the metadata, so interacting with UFS is not necessary. A value of 0
+will perform UFS sync for every single operation, and this is appropriate for when there are many
+files created or updated directly in the UFS (bypassing Alluxio). In this scenario, Alluxio will
+always query the latest state of the path from the UFS. Any time interval value is useful for to
+limit the frequency of UFS syncing. Larger values will perform UFS sync less, but the chances will
+be higher for the Alluxio metadata to be stale. Interacting with the UFS incurs performance
+overheads, so it is recommended to limit frequent UFS syncing.
+
+Since the UFS sync feature includes the loading metadata functionality, if UFS syncing is performed,
+the loading metadata configuration parameter, alluxio.user.file.metadata.load.type, is ignored. In
+comparison to the loading metadata feature, UFS sync takes care of any UFS modifications, like
+deleting files, or updating the timestamp, in addition to discovering new files.
+
+### Passive caching
```

Comment:
I remember that we decided not to talk about this @calvinjia , can you recall?

Reply:
Passive caching is something we hope to remove in the future.